### ISSUE TITLE
ERROR: Error(s) in loading state_dict for DPTDepthModel (Midas 3.1)

### ISSUE THREAD

**Issue by NicoSteyl** [thygate/stable-diffusion-webui-depthmap-script#427]
Trying to generate a depth map using Midas 3.1 results in an error, all the other models work fine. Only the 3.1 Midas models give this error:

SD Web UI 1.8.0

ERROR: Error(s) in loading state_dict for DPTDepthModel:
Unexpected key(s) in state_dict: "pretrained.model.blocks.0.attn.relative_position_index", "pretrained.model.blocks.1.attn.relative_position_index", "pretrained.model.blocks.2.attn.relative_position_index", "pretrained.model.blocks.3.attn.relative_position_index", "pretrained.model.blocks.4.attn.relative_position_index", "pretrained.model.blocks.5.attn.relative_position_index", "pretrained.model.blocks.6.attn.relative_position_index", "pretrained.model.blocks.7.attn.relative_position_index", "pretrained.model.blocks.8.attn.relative_position_index", "pretrained.model.blocks.9.attn.relative_position_index", "pretrained.model.blocks.10.attn.relative_position_index", "pretrained.model.blocks.11.attn.relative_position_index", "pretrained.model.blocks.12.attn.relative_position_index", "pretrained.model.blocks.13.attn.relative_position_index", "pretrained.model.blocks.14.attn.relative_position_index", "pretrained.model.blocks.15.attn.relative_position_index", "pretrained.model.blocks.16.attn.relative_position_index", "pretrained.model.blocks.17.attn.relative_position_index", "pretrained.model.blocks.18.attn.relative_position_index", "pretrained.model.blocks.19.attn.relative_position_index", "pretrained.model.blocks.20.attn.relative_position_index", "pretrained.model.blocks.21.attn.relative_position_index", "pretrained.model.blocks.22.attn.relative_position_index", "pretrained.model.blocks.23.attn.relative_position_index".

Please report this issue [here](https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues). Make sure to provide the full stacktrace:
Traceback (most recent call last):
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\src\common_ui.py", line 532, in run_generate
    input_i, type, result = next(gen_obj)
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\src\core.py", line 322, in core_generation_funnel
    raise e
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\src\core.py", line 124, in core_generation_funnel
    model_holder.ensure_models(inp[go.MODEL_TYPE], device, inp[go.BOOST])
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\src\depthmap_generation.py", line 62, in ensure_models
    self.load_models(model_type, device, boost)
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\src\depthmap_generation.py", line 124, in load_models
    model = DPTDepthModel(
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\dmidas\dpt_depth.py", line 163, in __init__
    self.load(path)
  File "G:\AI\stable-diffusion-webui-1.8.0\extensions\stable-diffusion-webui-depthmap-script\dmidas\base_model.py", line 16, in load
    self.load_state_dict(parameters)
  File "G:\AI\stable-diffusion-webui-1.8.0\venv\lib\site-packages\torch\nn\modules\module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for DPTDepthModel:
	Unexpected key(s) in state_dict: "pretrained.model.blocks.0.attn.relative_position_index", "pretrained.model.blocks.1.attn.relative_position_index", "pretrained.model.blocks.2.attn.relative_position_index", "pretrained.model.blocks.3.attn.relative_position_index", "pretrained.model.blocks.4.attn.relative_position_index", "pretrained.model.blocks.5.attn.relative_position_index", "pretrained.model.blocks.6.attn.relative_position_index", "pretrained.model.blocks.7.attn.relative_position_index", "pretrained.model.blocks.8.attn.relative_position_index", "pretrained.model.blocks.9.attn.relative_position_index", "pretrained.model.blocks.10.attn.relative_position_index", "pretrained.model.blocks.11.attn.relative_position_index", "pretrained.model.blocks.12.attn.relative_position_index", "pretrained.model.blocks.13.attn.relative_position_index", "pretrained.model.blocks.14.attn.relative_position_index", "pretrained.model.blocks.15.attn.relative_position_index", "pretrained.model.blocks.16.attn.relative_position_index", "pretrained.model.blocks.17.attn.relative_position_index", "pretrained.model.blocks.18.attn.relative_position_index", "pretrained.model.blocks.19.attn.relative_position_index", "pretrained.model.blocks.20.attn.relative_position_index", "pretrained.model.blocks.21.attn.relative_position_index", "pretrained.model.blocks.22.attn.relative_position_index", "pretrained.model.blocks.23.attn.relative_position_index". 

Time taken: 16.7 sec.

A: 2.09 GB, R: 2.11 GB, Sys: 3.2/12 GB (26.4%)

**Comment by SnowySailor** [comment]
Getting the same error. I see other people with this problem saying to just install `timm`, but I already have it and it's up to date as well.

**Comment by SnowySailor** [comment]
I actually fixed it by downgrading to 0.9.2 since I was on 0.9.16. however I run into a segfault when it gets to the computing outputs step:
```
auto-1  | loading the model from ./models/pix2pix/latest_net_G.pth
auto-1  | Computing output(s) ..
  0% 0/1 [00:00<?, ?it/s]wholeImage being processed in : 1536
auto-1  | Killed
auto-1 exited with code 137
```

Probably something unrelated though...

**Comment by semjon00** [comment]
Please launch webui with --disable-safe-unpickle command line flag
